.github/workflows/diff.yml

```diff
@@ -14,15 +14,25 @@ jobs:
     runs-on: ubuntu-latest
     container: obolibrary/odklite:v1.3.0
     steps:
+      - uses: khan/pull-request-comment-trigger@v1.1.0
+        id: check
+        with:
+          trigger: '#gogoeditdiff'
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
       - uses: actions/checkout@v3
+        if: steps.check.outputs.triggered == 'true'
       # Checks-out main branch under "main" directory
       - uses: actions/checkout@v3
+        if: steps.check.outputs.triggered == 'true'
         with:
           ref: master
           path: master
       - name: Diff classification
+        if: steps.check.outputs.triggered == 'true'
         run: export ROBOT_JAVA_ARGS=-Xmx6G; robot diff --labels True --left master/src/ontology/cl-edit.owl --left-catalog master/src/ontology/catalog-v001.xml --right src/ontology/cl-edit.owl --right-catalog src/ontology/catalog-v001.xml -f markdown -o edit-diff.md
```

Comment:
This does not cover changes to [components](https://github.com/obophenotype/cell-ontology/blob/master/src/ontology/cl-odk.yaml#L89). For the ODK version, it will be better to use the new _intermediate_ ontology which merges components into the edit file prior to further processing.